policy-engine/src/main.rs

```diff
@@ -96,6 +111,7 @@ fn main() -> Result<(), Error> {
     })
     .bind((settings.address, settings.port))?
     .start();
+    BUILD_INFO.inc();
```

Comment:
I'm curious, do you have a plan to use the counter with the build info for something?

Reply:
No, all we need is labels from this metric. `inc` here would be executed just once

Reply:
I see, so this is very similar to the `up` metric just with custom labels attached.